### ISSUE TITLE
How do bases and segments correspond one to one?

### ISSUE THREAD

**Issue by N-damo** [jmschrei/PyPore#1]

**Comment by jmschrei** [comment]
I'm not sure what you mean. A segment, which I think is now commonly referred to as an "event" in the literature, is a slice of signal that exhibits steady statistical parameters. In theory, each segment should correspond to a change of one nucleotide in the pore.

**Comment by N-damo** [comment]
A nanopore can hold k nucleotides (k-mer)
simultaneously. For example, k equals 5 for the pore version
R9.4. Thus, the current changes indicate the k-mers that pass
through the nanopores. I'm not sure whether each segment correspond to one nucleotide here.

**Comment by jmschrei** [comment]
Yes, in theory a segment should correspond to one nucleotide change in the pore. In this library, it's just a way to store a chunk of signal with stationary statistical properties. Whether the segments actually correspond to changes in the pore depends on the quality of the segmentation algorithm.

**Comment by N-damo** [comment]
everything is amazing. In the supervised machine learning, back to the very beginning, people how to know raw signal correspond to a real sequence.
